apps/accounts/forms/auth.py

```diff
@@ -56,17 +56,26 @@ class UpdatePasswordForm(SetPasswordForm):
     """A subclass of Django's SetPasswordForm
     Sends out a password_changed_email
     """
+
     def __init__(self, *args, **kwargs):
         super(UpdatePasswordForm, self).__init__(*args, **kwargs)
         self.label_suffix = ''
         set_input_placeholder_labels(self)
         set_input_attrs(self)
 
-    def save(self, commit=True):
+    def save(
+        self,
+        request=None,
+        email_template=None,
+        email_subject=None,
+        email_sender=None,
```

Comment:
These two params aren't being used in the function, are they? If not, don't include them now.

Or if you include them, implement the logic.